packages/functions/index.d.ts

```diff
@@ -14,6 +14,4 @@
  * limitations under the License.
  */
 
-export { createEditBatch } from "./createEditBatch.js";
-export type { EditBatch } from "./EditBatch.js";
-export type { Edits } from "./types.js";
+export * from "./build/cjs/index.cjs";
```

Comment:
Its because of the way `node10` style commonjs module resolution works and while node isn't doing this given our setup, typescript in default config does look at this like the silly dog that it is.